box/box_test.go

```diff
@@ -1,15 +1,19 @@
-package box_test
+package box
```

Comment:
```suggestion
package box_test
```

Nit: please, test only the public API.